### PR TITLE
[aws-dynamodb] Adjust options and graph definitions

### PR DESCRIPTION
**Pull request by astj**
- Remove `-tempfile` option, since this plugin does not use tempfile (= does not have metrics with `Diff: true`)
- Declare operational metrics explicitly (, not wildcard) in Graph definitions
  - inactive metrics with wildcard graph definitions will be deleted
  - ref: https://mackerel.io/api-docs/entry/host-metrics#post-graphdef

### CODE REVIEW

**Comment by stefafafan** [comment]
The changes look good to me, just wondering about metrics that do not seem to be included in the plugin (e.g. `OnlineIndexConsumedWriteCapacity` ).

**Comment by astj** [comment]
Since we don't use global secondary index in our project, I omitted metrics which needs global secondary index in #349.
If someone needs I'll implement them in another p-r.

File: mackerel-plugin-aws-dynamodb/lib/aws-dynamo-db.go [hunk 1]
```diff
@@ -321,35 +321,70 @@ func (p DynamoDBPlugin) GraphDefinition() map[string]mp.Graphs {
 			Label: (labelPrefix + " ThrottledRequests"),
 			Unit:  "integer",
 			Metrics: []mp.Metrics{
-				{Name: "*", Label: "%1", Stacked: true},
+				{Name: "ThrottleRequests.PutItem", Label: "PutItem", Stacked: true},
```
Comment (stefafafan) [inline mackerel-plugin-aws-dynamodb/lib/aws-dynamo-db.go]: Should these be `ThrottledRequests` rather than `ThrottleRequests` (no `d`) ?

Comment (astj) [inline mackerel-plugin-aws-dynamodb/lib/aws-dynamo-db.go]: Right. Fixed: https://github.com/mackerelio/mackerel-agent-plugins/pull/375/commits/1c464fd6be4b0afcac674249c69df1ead2c6598b

Comment (stefafafan) [inline mackerel-plugin-aws-dynamodb/lib/aws-dynamo-db.go]: Thanks 👍

**Comment by stefafafan** [comment]
Looks good to me!

**Comment by astj** [comment]
Thank you!

**Pull request merged**